.github/workflows/pr-merge-conflict.yml

```
@@ -0,0 +1,92 @@
+# This action is centrally managed in https://github.com/asyncapi/.github/
+# Don't make changes to this file in this repo as they will be overwritten with changes made to the same file in above mentioned repo
+
+# This action handles merge conflicts and provides auto-rebase functionality for pull requests
+name: Merge Conflict and Auto-Rebase 
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, edited, ready_for_review]
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-merge-conflicts:
+    if: github.event_name == 'pull_request_target'
+    runs-on: ubuntu-latest
+    outputs:
+      conflicts: ${{ steps.conflicts.outputs.conflicts }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v4
```

always latest please, and use version when official GH action, in case of community actions do not use version but point to commitId